### PR TITLE
Update status in upper right of card

### PR DESCRIPTION
**Pull request by amirpourpanahi**
### Description

List of proposed changes:

- Update latest status part in upper right of card header

### CODE REVIEW

File: locales/en.js [hunk 1]
```diff
@@ -319,6 +319,9 @@ export default {
     'https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada.html',
   url_myBenefitsAndServices: '/dashboard',
 
+  // Benefit card header URLs
+  url_ststusAndMessages: '/dashboard',
```
Comment (s-laugh) [inline locales/en.js:323]: property has a typo (will need to fix where used)
```suggestion
  url_statusAndMessages: '/dashboard',
```

Comment (amirpourpanahi) [reply locales/en.js:323]: Fixed

File: components/atoms/BenefitCardHeaderActive.js [hunk 2]
```diff
@@ -68,10 +68,16 @@ export default function BenefitCardHeaderActive(props) {
...
             className="sm:pb-2"
           >
-            <p className="font-display sm:pb-2">{t.latestUpdate}</p>
+            <p className="font-display sm:pb-2">{t.latestStatus}</p>
             <p className="font-display font-bold text-lg">
               {props.benefit.benefitStatusProgress}
             </p>
+            <a
+              href={t.url_statusAndMessages}
+              className="text-sm mt-1 text-bright-blue-solid underline"
```
Comment (s-laugh) [inline components/atoms/BenefitCardHeaderActive.js:77]: The `text-sm` shouldn't be used (brings the font to 14px, when the min should be 16px)
```suggestion
              className="mt-1 text-bright-blue-solid underline"
```

File: components/atoms/BenefitCardHeaderPending.js [hunk 3]
```diff
@@ -53,11 +53,21 @@ export default function BenefitCardHeaderPending(props) {
...
+            <p className="font-display sm:pb-2">{t.latestStatus}</p>
             <p className="font-display font-bold text-lg">
               {props.benefit.benefitStatusProgress}
             </p>
+            <p className="font-bold text-lg">
+              {props.benefit.latestStatusDate}
+            </p>
+            <a
+              href={t.url_statusAndMessages}
+              className="text-sm mt-1 text-bright-blue-solid underline"
```
Comment (s-laugh) [inline components/atoms/BenefitCardHeaderPending.js:67]: Same thing
```suggestion
              className="mt-1 text-bright-blue-solid underline"
```

**Review by s-laugh — CHANGES_REQUESTED** [review]
Sorry I didn't catch this in my first look

File: components/atoms/BenefitCardHeaderActive.js (hunk 2 above)
Comment (amirpourpanahi) [reply components/atoms/BenefitCardHeaderActive.js]: Fixed, thanks

**Review by nabil386 — APPROVED** [review]

**Review by s-laugh — APPROVED** [review]
Great!

**Pull request merged**